scripts/src/chartrepomanager/chartrepomanager.py

```diff
@@ -401,204 +311,6 @@ def create_index_from_report(category, ocp_version_range, report_path):
     return chart_entry
 
 
-def set_package_digest(chart_entry):
```

Comment:
I might be missing it - is this being calculated and added in the replacement logic that handles index updates?